homework1/homework1.css

```diff
@@ -0,0 +1,17 @@
+* {margin: 0; padding: 0;}
```

Comment:
Shouldn't use this selector (Không nên dùng css selector kiểu này) if we don't use any css framework, i.e Boostrap, Tailwind (Nếu mình ko dùng CSS framwork nào như là [boostrap](https://getbootstrap.com/docs/3.4/css/) hay tailwind, bơi vì mặc định các thẻ html đang ko có margin hay padding)

Reply:
thank you brother. i will remember it